timescaledb/getting-started/add-data.md

```diff
@@ -40,17 +38,27 @@ includes a separate table of company symbols and company names, in a table named
     );
     ```
 
-1.  Download the following `.zip` file. The file contains two `.csv` files: one with company information, and one with real-time stock trades for one month.
+1.  Download the following `.zip` file. The file contains two `.csv` files: one with company 
+information, and one with real-time stock trades for one month.
 
-    Download: <tag type="download">[stock_data_real_time.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/)</tag>
+    Download: <tag type="download">[real_time_stock_data.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/)</tag>
 
-1.  At the command prompt, unzip the `.csv` files:
+1.  In a new terminal window, run this code to unzip the `.csv` files:
     ```bash
     unzip stock_data_real_time.zip
     ```
 
-1.  At the `psql` prompt, insert the data into your 
-    TimescleDB instance. If the `.csv` files aren't in your current directory, replace the file paths as needed:
+1.  For folks using Docker containers, you will have to add the data files to your container before 
+copying them into your database:
+    ```bash
+    docker cp stocks_real_time.csv timescaledb:/stocks_real_time.csv
+
+    docker cp company.csv timescaledb:/company.csv
+    ```
+
+
+1. At the `psql` prompt, insert the data into your TimescleDB instance. If the `.csv` files aren't in 
+your current directory, replace the file paths as needed:
```

Comment:
The numbering is resetting here:

![image](https://user-images.githubusercontent.com/3914967/159819655-d5e84356-3441-47ed-8e3f-01a369f6d022.png)